.github/pull_request_template.md

```diff
@@ -22,6 +22,9 @@
 
 - [ ] I have run the complete test suite to ensure all tests and linters pass.
 - [ ] I have thoroughly tested my code changes to ensure they work as expected, including adding additional tests for new functionality.
+- [ ] I have run Sorbet type-checking in strict mode to ensure that all types are correct and complete.
 - [ ] I have written clear and descriptive commit messages.
 - [ ] I have provided a detailed description of the changes in the pull request, including the problem it addresses, how it fixes the problem, and any relevant details about the implementation.
 - [ ] I have ensured that the code is well-documented and easy to understand.
+- [ ] I have ensured that there are no breaking changes unless explicitly stated and handled.
+- [ ] I have checked the backward compatibility of my changes.
```

Comment:
I think the intent of this is great, but I don't think this is a realistic expectation for our drive-by open source contributors.

IMO this should at least partially be the responsibility of the maintainers to eyeball for compatibility.

Instead, I'd word this as a reminder that "if you're aware of 🦶 🔫 , please call it out"

```suggestion
- [ ] I have called out any breaking changes that I'm aware of.
```